applications/albaspot/README.md

```diff
@@ -0,0 +1,69 @@
+```
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 Albora Technologies Ltd.
+```
+
+# Albora Overview
+
+Albora Technologies develops next-generation high-accuracy geolocation
+solutions for industrial and mass-market applications. The company's
+technologies offer fast, high-accuracy, software-based and
+hardware-agnostic GNSS solutions, enabling customers to obtain better
+geolocation information that is reliable, safe and secure. Our products
+are especially suited for tracking assets in challenging environments,
+such as urban areas or industrial/infrastructure complexes, where
+certainty of the asset location is important.
+
+Founded in 2017, the company is headquartered in London, United
+Kingdom, and has an office in Barcelona, Spain.
+
+For more information on Albora Technologies, please visit our
+website: [Albora Technologies](https://albora.io)
+
+# AlbaSpot Overview
+
+AlbaSpot is an affordable geolocation edge/cloud-based corrections service
+that provides continuous high-accuracy positioning. Its features include:
+
+* High-accuracy: centimetre-level accuracy in real time
+* Multi-constellation: supports GPS, Galileo, Beidou aand Glonass
+* Multi-frequency: supports L1, L2 and L5
+* Customizable: processing can be ported across edge, cloud and private data centers
+* Standardized: compatible with standard communication protocols (NTRIP/RTCM)
+* Fast and reliable: low latency due to optimized network design
+* Hardware agnostic: supports all third-party NTRIP/RTCM-compatible devices in the market
+
```

Comment:
Hi,
Could you please add this section-  https://intel-https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#pre-requisites---installing-openness. If I remember clearly OpenNESS version used is 21.03, flavor used is single node minimal and Distribution is OpenSource.

Reply:
Done, additional information added.

Reply:
Thanks. I will have to request you to please remove the IP from yaml file. You can mark it as xxx.xx.xx.xx if you want.

Reply:
Thanks for the comment! I've removed the IP addresses as requested.